modules/pricer/src/test/java/com/opengamma/strata/pricer/impl/swap/DiscountingFxResetNotionalExchangePricerTest.java

```diff
@@ -186,6 +189,60 @@ public void test_explainPresentValue_paymentDateInPast() {
   }
 
   //-------------------------------------------------------------------------
+  public void test_currencyExposure() {
```

Comment:
Test currency exposure by finite difference: compare change of value of the product with change of value of the MultiCurrencyAmount currency exposure when the underlying exchange rate in FX_MATRIX is changed by a small amount.
The test should be done for a FxResetNotional which as not fixed and one which has fixed (i.e. the valuation date is between fixing and payment).